### DISCUSSION TITLE
Welcome to blackarch Discussions!

### DISCUSSION THREAD

**Discussion by noptrix** [BlackArch/blackarch#3020] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Youssef-Tamer** [comment]
The menus of blackarch don't show me after installing kde plasma there is any solution?
and there is some bugs like there is some text don't show in any window or applications in kde plasma 
There is any solutions?

**Reply by Edu4rdSHL** [reply]
Hello, this kind of problems should be opened as issue, not here. I have created #3021

**Reply by Youssef-Tamer** [reply]
Okay thank you.. #3021
